src/plrust.rs

```diff
@@ -93,7 +70,7 @@ pub(crate) unsafe fn evaluate_function(
 }
 
 #[tracing::instrument(level = "debug")]
-pub(crate) fn compile_function(fn_oid: pg_sys::Oid) -> eyre::Result<(PathBuf, Output)> {
+pub(crate) fn compile_function(fn_oid: pg_sys::Oid) -> eyre::Result<Output> {
```

Comment:
Hm. Were we really ignoring this PathBuf at every call site? At first it feels weird but I guess it's fair to clean it up, yeah.

Reply:
We weren't, but we are now since the function is stored in the database.